### ISSUE TITLE
[Core] Add keyboard controls to the section selector in upper-left of core pages

### ISSUE THREAD

**Issue by Catacola** [Recidiviz/pulse-dashboard#919]
**Is your feature request related to a problem? Please describe.**
The nav bubble in core pages isn't navigable by keyboard and it should be.

**Describe the solution you'd like**
Arrows to change selection. Enter to select. Esc to leave.

**Additional Context**
This can easily be done by autofocusing the first `DropdownItem` in `CoreSectionSelector`, but it leads to ugly design state where an element is always highlighted.
